quranHome/src/main/res/drawable/ic_bg_juz_cirle_start.xml

```diff
@@ -0,0 +1,11 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I think we can use a single drawable for both start and end.
Tint we can change their color when ever needed